stable/redis/templates/redis-master-statefulset.yaml

```diff
@@ -172,10 +166,8 @@ spec:
     - metadata:
         name: redis-data
         labels:
-          app: "{{ template "redis.name" . }}"
-          component: "master"
-          release: {{ .Release.Name | quote }}
-          heritage: {{ .Release.Service | quote }}
+          app.kubernetes.io/component: server
+{{ include "redis.instance-name" . | indent 10 }}
```

Comment:
This drops `heritage`. Do we want to include `app.kubernetes.io/managed-by` here?

Reply:
volumeClaimTemplates are a little touchy on labeling. It makes sense to remove the managed-by here just like it shouldn't be in matchLabels.